src/applications/vaos/services/appointment/transformers.js

```diff
@@ -287,13 +287,14 @@ function setVideoData(appt) {
       : null,
     atlasConfirmationCode: videoData.tasInfo?.confirmationCode,
     duration: videoData.duration,
-    status: videoData.status.code,
   };
 }
 
 function getCommunityCareData(appt) {
   if (!isCommunityCare(appt)) {
-    return {};
+    return {
```

Comment:
I made some changes in this transformer to explicitly return null values in most places, since it makes it easier to be consistent with v2 data.